ogc-example/src/main/scala/geotrellis/server/ogc/conf/Conf.scala

```diff
@@ -11,15 +11,14 @@ import pureconfig.ConfigReader
 import scalaxb.DataRecord
 
 case class Conf(
-  http: Conf.Http,
   layers: Map[String, OgcSourceConf],
   wms: Conf.WMS,
   wmts: Conf.WMTS,
   wcs: Conf.WCS
 )
 
 object Conf {
-  trait OgcService {
+  trait OgcServiceConf {
```

Comment:
Btw why these classes are inside of the object? If users would use them it would be really not convenient to call them.